js/ui/applet.js

```diff
@@ -377,9 +377,14 @@ Applet.prototype = {
             this.actor.set_important(true);
             this.actor.set_vertical(true);
             this.actor.set_x_expand(true);
+
+            this.actor.set_clip(0,0,this._panelHeight,5000); // ensure no portion of the applet
```

Comment:
This should have spaces between the arguments (0, 0, this._panelHeight, 5000)
If you just put the comment above inline
```
// ensure no portion...
this.actor.set_clip()
```
it would all fit in one line. If you're going to put them after the line they are probably better as one single line and not as a multi-line comment.